src/controllers/teamController.js

```diff
@@ -70,6 +70,15 @@ const teamcontroller = function (Team) {
         res.status(400).send('No valid records found');
         return;
       }
+
+      const canEditTeamCode = req.body.requestor.role === 'Owner'
```

Comment:
Nice work.

But it will be nice to have the role property value to be in all lowercase letters. For example: `role = 'owner' `

This will help with consistency and reduce potential bugs that are related to different case letters.

I highly recommend that the use of lowercase letters should also reflect in the other possible role values.

Kindly review.

Reply:
Thanks for your suggestion! But this change involves a large number of files. I would say it's better to add this request to the bug file and create a new PR for it.